src/main.c

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Okay. Is `main.c` is a whole new file, not a derivation of the `main.cpp`? Just curious.